### DISCUSSION TITLE
Using Digger with trunk based development

### DISCUSSION THREAD

**Discussion by mathieux51** [diggerhq/digger#1266] [Q&A]
Howdy folks, 
Is it possible to use Digger with [Trunk Based Development](https://trunkbaseddevelopment.com/), so no pull requests, only direct pushes to the main branch?
No sure who to ping, maybe @motatoes would know

**Comment by UtpalJayNadiger** [comment]
We're actively working on this! Would love your feedback on what we've shipped so far. Any chance you'd be open to a [call](https://calendly.com/diggerdev/community-feedback) some time this week?

**Reply by mathieux51** [reply]
Sure, call scheduled 😇

**Comment by boldandbusted** [comment]
Any news? I'd love to see this. :fireworks: :bow: :fireworks:
